apps/cookbook/src/app/examples/slides-example/slides-example.shared.scss

```diff
@@ -0,0 +1,7 @@
+@use '@kirbydesign/core/src/scss/utils';
+@use '../examples.shared';
+
+:host {
+  --padding-start: #{utils.size('s')};
+  --padding-end: #{utils.size('s')};
+}
```

Comment:
I don't understand why these are needed - are they used for anything right now?

Reply:
Yes, it's what makes the Slides stretch to the viewport (we use them for negative margins to counterfeit the padding of the page):
![image](https://github.com/kirbydesign/designsystem/assets/9556693/715a031a-a440-409a-9c5c-03c58c42de5a)

Reply:
Ah alright, and because these are set by page normally, it works as expected?

Reply:
Yes exactly 👍🏻